### DISCUSSION TITLE
What are grant_type, scope, client_id, and client_secret parameters in /auth/login route body for?

### DISCUSSION THREAD

**Discussion by NickNaskida** [fastapi-users/fastapi-users#1305] [Q&A] [answered]
Hey, I have been using this framework for a while.

I can't find documentation for these body parameters `grant_type`, `scope`, `client_id`, and `client_secret`. What is their purpose? They seem to be missing in documentation

![CleanShot 2023-10-19 at 12 11 30@2x](https://github.com/fastapi-users/fastapi-users/assets/82929931/da6856b5-73bd-446e-a2a8-a7849501c556)

**Comment by frankie567** [comment, marked as answer]
Hi! 

Those parameters actually come from the FastAPI utility managing the authentication for Swagger.

In our case, they are unused since we are not implementing a full OAuth2 authentication.

**Reply by NickNaskida** [reply]
Got it, thanks!
